code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -781,14 +781,14 @@
 	hitsound_on = 'sound/weapons/bladeslice.ogg'
 	w_class = WEIGHT_CLASS_BULKY
 	sharpness = SHARP_EDGED
-	faction_bonus_force = 30
+	faction_bonus_force = 5
```

Comment:
Not necessary, the faction bonus only applies to simple mobs.
If you really want to nerf the saw that much - make it 15-20.